eZ/Publish/API/Repository/Repository.php

```diff
@@ -185,6 +185,8 @@ public function getUserPreferenceService(): UserPreferenceService;
      *
      * Begins an transaction, make sure you'll call commit or rollback when done,
      * otherwise work will be lost.
+     *
+     * @deprecated since eZ Platform 3.3.0, to be removed in eZ Platform 4.0.0. Use \eZ\Publish\API\Repository\TransactionService::beginTransaction instead.
```

Comment:
We need to use here Ibexa 3.3.0 instead of eZ Platform.

Reply:
Fixed in https://github.com/ezsystems/ezplatform-kernel/pull/164/commits/ffb35ff9c0bd6d94b14a1401cfb71a4e9312cb44